python/pyarrow/tests/parquet/test_multifile_ds.py

```diff
@@ -0,0 +1,1662 @@
+# Licensed to the Apache Software Foundation (ASF) under one
```

Comment:
Maybe we can call this one `test_dataset.py` ?

Reply:
Done